source/guides/network/wifi.rst

```diff
@@ -0,0 +1,99 @@
+.. _wi-fi:
+
+Configure Wi-Fi with NetworkManager
+###################################
+
+We recommend using `NetworkManager <https://developer.gnome.org/NetworkManager/stable/NetworkManager.html>`_ to manage network connections. If you choose to
+connect to Wi-Fi while using the live image, your Wi-Fi settings will be
+added to your system during the installation process.
+
+NetworkManager provides three simple methods for configuring Wi-Fi.
+
+.. contents::
+   :local:
+   :depth: 1
+
+Desktop GUI (Graphical User Interface)
+**************************************
+
+1. Click anywhere on the icons at the right side of the top of the screen to
+   bring up a menu and click on :guilabel:`Wi-Fi Not Connected` then
+   choose :guilabel:`Select Network`.
+
+   .. figure:: /_figures/wifi/wifi-1.1.png
```

Comment:
Suggest cropping this into the upper right corner of the GNOME desktop. The full desktop screenshot is a bit much.